### PR TITLE
Es 10 create poi

### PR DESCRIPTION
**Pull request by r4sinhas**

### CODE REVIEW

**Review by GoncaloSilva25 — APPROVED** [review]
Good coverage

File: app/components/MarkersManager.tsx [hunk 1]
```diff
@@ -2,69 +2,313 @@ import { Marker, Popup, useMapEvents } from "react-leaflet";
...
+    if (name === "" || type === "") {
+      closePOIModal();
+      return;
+    }
+
+    // validate that name and type are not too long
+    else if (name.length > 20 || type.length > 20) {
+      closePOIModal();
+      return;
+    }
```
Comment (joaompfonseca) [inline app/components/MarkersManager.tsx:80]: Why not display a message to the user instead of closing the model?

Comment (r4sinhas) [reply app/components/MarkersManager.tsx:80]: had an alert (testing only), mas dps apaguei.
Will implement a form control, ty for the reminder :)

**Review by joaompfonseca — APPROVED** [review]
LGTM

**Pull request merged**